### PR TITLE
Add docs for Groq and for OpenAI proxies

### PR DESCRIPTION
**Pull request by enyst**
**CHANGELOG**

Just some docs, so we have clear cases what works:
- using provider groq
- using openai proxies

---

@mamoodi TBH Groq is a little strange to add along the other "llms", since it has no models of its own, it just serves [a number of models](https://console.groq.com/docs/models) - presumably to showcase how fast their chips are. But maybe it's better to have a few notes documented than be unsure how it works?

### CODE REVIEW

File: docs/modules/usage/llms/openai-llms.md [hunk 1]
```diff
@@ -14,3 +14,10 @@ If the model is not in the list, toggle `Advanced Options`, and enter it in `Cus
 ## Using OpenAI-Compatible Endpoints
 
 Just as for OpenAI Chat completions, we use LiteLLM for OpenAI-compatible endpoints. You can find their full documentation on this topic [here](https://docs.litellm.ai/docs/providers/openai_compatible).
+
+## Using an OpenAI proxy
```
Comment (tobitege) [inline docs/modules/usage/llms/openai-llms.md:18]: Tiny nit: "Proxy" :)

**Review by tobitege — APPROVED** [review]
LGTM, thanks!

**Comment by mamoodi** [comment]
That's fine @enyst. I think the whole "llms" section is kind of weird and we should probably restructure it a bit. The others are also providers so this is great for now and then we can see what to do.

File: docs/modules/usage/llms/groq.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Running LLMs on Groq
+
+OpenHands uses LiteLLM to make calls to chat models on Groq. You can find their full documentation on using Groq as provider [here](https://docs.litellm.ai/docs/providers/groq). When running OpenHands, it's simpler: you only need to set the options below.
```
Comment (mamoodi) [inline docs/modules/usage/llms/groq.md:3]: Can you move the last sentence to the Configuration section please so it lines up with how we do other ones?

"When running OpenHands, you'll need to set the following in the OpenHands UI through the Settings:
...
"

**Comment by mamoodi** [comment]
Thank you for this!

**Review by mamoodi — APPROVED** [review]

File: docs/modules/usage/llms/groq.md (hunk 2 above)
Comment (enyst) [reply docs/modules/usage/llms/groq.md]: I adjusted it, still kept a little before. I just wanted to suggest somehow that the litellm docs we just linked to may be useful, but not everything in them is necessary, we literally make it simpler (e.g. no need for env vars of specific names)

File: docs/modules/usage/llms/groq.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# Running LLMs on Groq
+
+OpenHands uses LiteLLM to make calls to chat models on Groq. You can find their full documentation on using Groq as provider [here](https://docs.litellm.ai/docs/providers/groq). With OpenHands, it's simpler: please see the configuration below.
```
Comment (enyst) [inline docs/modules/usage/llms/groq.md:3]:
```suggestion
OpenHands uses LiteLLM to make calls to chat models on Groq. You can find their full documentation on using Groq as provider [here](https://docs.litellm.ai/docs/providers/groq).
```

File: docs/modules/usage/llms/groq.md (hunk 2 above)
Comment (enyst) [reply docs/modules/usage/llms/groq.md]: Actually, I changed my mind and removed that extra bit. It lines up, yes.

**Pull request merged**